src/game.cpp

```diff
@@ -120,6 +120,18 @@ bool is_valid_in_w_terrain(int x, int y)
     return x >= 0 && x < TERRAIN_WINDOW_WIDTH && y >= 0 && y < TERRAIN_WINDOW_HEIGHT;
 }
 
+/*
+ * case insensitive string::find( string::findstr ). findstr must be lowercased
+ */
+bool lcmatch(const std::string &str, const std::string &findstr)
```

Comment:
because I moved declaration to game.h so every second .cpp file would stop redeclare it. And since it declared in game.h, might be defined in game.cpp as well.